ledger-core/v2/virtual/execute/execute.go

```diff
@@ -301,13 +301,14 @@ func (s *SMExecute) stepStartRequestProcessing(ctx smachine.ExecutionContext) sm
 	var objectDescriptor descriptor.Object
 
 	action := func(state *object.SharedState) {
-		if _, ok := state.KnownRequests[s.execution.Outgoing]; ok {
+		requestList := state.KnownRequests.GetList(s.execution.Isolation.Interference)
+		if requestList.Exist(s.execution.Outgoing) {
```

Comment:
Next line `requestList.Add(s.execution.Outgoing)` will return `false` if such record exist. You can combine add and check in one condition

Reply:
good point, will fix in next PR >_< missed your comment